tools/floodgate/js/floodgate.js

```diff
@@ -15,6 +15,8 @@ import {
 import promoteFloodgatedFiles from './promote.js';
 import floodgateContent from './copy.js';
 
+const modal = document.getElementById('fg-modal');
```

Comment:
You should avoid putting variables in the global scope. If `modal` is only needed in `setListeners`, then it should live there